isthmus/src/test/java/io/substrait/isthmus/TpcdsQueryNoValidation.java

```diff
@@ -20,4 +29,25 @@ public void tpcds(int query) throws Exception {
     var plan = s.execute(asString(String.format("tpcds/queries/%02d.sql", query)), creates);
     System.out.println(JsonFormat.printer().print(plan));
   }
+
```

Comment:
Can we remove the original `tpcds` testing function as it is no longer needed?
In addition, when removing this function the partial [`schema.sql`](https://github.com/substrait-io/substrait-java/blob/main/isthmus/src/test/resources/tpcds/schema.sql) definition can be removed as well.

Reply:
Moved the test to a new file: `KeyConstraintsTest`